templates/test/ci/cluster-template-prow-ci-version-windows-containerd-2022.yaml

```diff
@@ -1095,7 +1098,7 @@ data:
             - --kubelet-use-node-status-port
             - --metric-resolution=15s
             - --kubelet-insecure-tls
-            image: k8s.gcr.io/metrics-server/metrics-server:v0.5.2
+            image: k8s.gcr.io/metrics-server/metrics-server:v0.6.2
```

Comment:
```suggestion
            image: registry.k8s.io/metrics-server/metrics-server:v0.6.2
```

Reply:
these files are all auto-generated from https://github.com/kubernetes-sigs/metrics-server/releases/download/v0.6.2/components.yaml which still uses `k8s.gcr.io`

Reply:
D'oh sorry, I knew that. So we have to wait for that external resource to be updated to actually close #3035?

Reply:
yes